Refs #31626 -- Noted that Daphne provides a runserver command. (#16634)
django/django: docs/howto/deployment/asgi/daphne.txt

This will start one process listening on ``127.0.0.1:8000``. It requires that
your project be on the Python path; to ensure that run this command from the
same directory as your ``manage.py`` file.

Integration with ``runserver``
==============================

> Daphne also provides a :djadmin:`runserver` command to serve your site with
> ASGI during development.

Reviewer: I would remove `also` :thinking: Maybe:
```suggestion
Daphne provides a :djadmin:`runserver` command to serve your site under ASGI
during development.
```

This can be enabled by adding ``daphne`` to the start of your
:setting:`INSTALLED_APPS` and adding an ``ASGI_APPLICATION`` setting pointing
your ASGI application object::

    INSTALLED_APPS = [
        "daphne",
        ...,
    ]

>     ASGI_APPLICATION = "myproject.asgi.application"

Reviewer: What do you think about adding a short section to `runserver` [docs](https://docs.djangoproject.com/en/4.1/ref/django-admin/#runserver)? For example:
```
Lack of built-in ASGI support
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

The ``runserver`` command doesn't support ASGI, however, there are 3rd-party
packages that provide the ``runserver`` (or similar) command to serve your site
under ASGI during development. See ... for details.

```

Author: OK, I pushed edits. Let me know what you think.
